### ISSUE TITLE
VS 2022 - not adding to Extensions or right-click menus

### ISSUE THREAD

**Issue by Andy-Corrigan-Liberis** [codecadwallader/codemaid#999]
### Environment
- Visual Studio version: 2022 Community (latest update)
- CodeMaid version: latest
- Code language: Inconsequential to the issue at hand

### Description
Since the latest Visual Studio update, it seems that CodeMaid (and other extensions) are unable to add to the in-built VS menus. I've raised an issue with Microsoft as other extensions are affected (it's just that yours relies solely on adding to VS menus, and is therefore unusable); I'm pretty sure the issue is with VS (or, at the very least, my install) - but they seem convinced it's not. For reference, the issue (which has screenshots) is here: https://developercommunity.visualstudio.com/t/Manage-Extensions-not-installing-extensi/10359776

### Steps to recreate
1. Install extension
2. Re-open VS (any project/repo)
3. Try to access the extension

### Current behavior
Extension tools not visible

### Expected behavior
Extension tools should be accessible

**Comment by Andy-Corrigan-Liberis** [comment]
Update: Microsoft have decided they don't have the capacity to look into this.

**Comment by codecadwallader** [comment]
Thanks for reporting the issue.  I have not been able to reproduce it.  There can be a delay in menus populating as extensions are asynchronously loaded and delayed until a solution is opened.  However if you have a solution open and the Visual Studio status bar appears to be finished loading everything I wouldn't expect the menu to not be populated.

**Comment by Andy-Corrigan-Liberis** [comment]
Yeah, I suspected it was something to do with my laptop/VS install - even getting an error just doing Ctrl+C 😆

**Comment by wh1sp3r** [comment]
Hi,
I have same issue. After VS2022 update to 17.7.1... all context menu for Codemaid disappeared.
I reinstalled VS2022 and extensions and there are not visible.
It also seems other extensions also disappeared, so it's not a Codemaid problem. Any idea? Thanks!

**Comment by LouisAtGH** [comment]
Same problem here!

I am running Blend for visual studio and installed e.g. the php extension DevSense. It shows up as installed extension but it is not accessible / visible in the menu's

In fact the situation is more complex:
- If I run Blend as user => extensions not accessible
- If I run Blend as Administrator from the user account => extensions not accessible
- If I login as Administrator and start Blend, the (just installed as user) extension is OK  (and there are more extensions) !!!

Desperately, I reinstalled Blend via the installer menu. That works however ......
it does not change any thing in the behavoir.

I plan to test via an other user account, however .... terrible ....  !!
Costing hours .... and still not working

**Comment by LouisAtGH** [comment]
I just logged in as another user ..... does not work as well .. :(
